File: fibonacci.c
#include<stdio.h>
int main()
{
    int n;
    printf("Enter the number of terms you want:");
    scanf("%d",&n);
    int i, a=0, b=1, c;
    printf("The required fibonacci series is:\n");
    printf("%d\t",a);
    printf("%d\t",b);
    for(i=0; i<=n; i++)
    {
        
        a=b;
        b=c;
        c=a+b;
        printf("%d\t",c);
    }
    
    return 0;
}

File: fibonacci.py
def fibs(n):
    n = int(input("Enter no of terms you want:"))
    a = 0
    b = 1
    
    print(a, end=" ")
    print(b, end=" ")

    for i in range(0, n-2):
        c = a+b
        a=b
        b=c
        print(c, end=" ")

# Make sure there are absolutely NO spaces before this line!
fibs(5)


    